### PR TITLE
Stop using incumbent settings object

### LINKED ISSUE #60: Stop using the "incumbent settings object"
See https://github.com/whatwg/html/issues/1430 for background.

### PR DESCRIPTION
**Pull request by domenic**
Fixes #60. Also adds a warning about #63.

I was unable to generate the index.html due to some macro REPOSITORYURL not being present. As such I'm not sure the warning stuff turned out all right. Maybe it would be a good idea to add instructions or a Makefile.

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -1677,9 +1677,14 @@ Origin of Blob URLs</h4>
 <a>Blob URLs</a> are created using <code>URL.{{URL/createObjectURL()}}</code>,
 and are revoked using <code>URL.{{URL/revokeObjectURL()}}</code>.
 The <dfn lt="origin of a Blob URL|Blob URL's origin">origin of a Blob URL</dfn> must be the same as the [=environment settings object/origin=]
-specified by the <a>incumbent settings object</a>
+specified by the <a>current settings object</a>
 at the time the method that created it was called.
 
+WARNING: there is currently some confusion between the generic definition of the
```
Comment (mkruisselbrink) [inline index.bs]: I don't think "WARNING" is a thing with special meaning in bikeshed. Probably `Advisement:` or `Issue:` makes the most sense?

**Comment by mkruisselbrink** [comment]
> I was unable to generate the index.html due to some macro REPOSITORYURL not being present. As such I'm not sure the warning stuff turned out all right. Maybe it would be a good idea to add instructions or a Makefile.

I'm curious what error you're getting? As far as I know I'm not doing anything special/not special configuration should be needed...

**Comment by domenic** [comment]
> I'm curious what error you're getting? As far as I know I'm not doing anything special/not special configuration should be needed...

When I do

```
curl https://api.csswg.org/bikeshed/ -F file=@index.bs > index.html
```

I get an index.html that contains

```
Error running preprocessor, returned code: 1. \033[1;31mFATAL ERROR:\033[0m Found unmatched text macro [REPOSITORYURL]. Correct the macro, or escape it with a leading backslash. \033[7;31m âœ˜ \033[0m Did not generate, due to fatal errors
```

**Comment by mkruisselbrink** [comment]
Ah, it's probably the fact that there is no "Repository:" metadata listed, instead relying on it being run from within a github checkout. Not sure why bikeshed even lets you do that...

**Comment by mkruisselbrink** [comment]
Thanks!

**Pull request merged**